apps/src/templates/sectionsRefresh/QuickAssignTable.jsx

```diff
@@ -28,55 +29,16 @@ export default function QuickAssignTable({
           </tr>
         </thead>
         <tbody className={moduleStyles.tableBody}>
-          {renderRows(courseOfferings[marketingAudience][key])}
+          {renderRows(
+            courseOfferings[marketingAudience][key],
+            sectionCourse,
```

Comment:
Because this is now happening in the shared helper file, we need to send in the sectionCourse so we know the current assignment, and the ability to update the section when a new radio is selected.